blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,267 @@
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+The overarching experience of generating diffs, creating and reviewing a campaign, then managing the changes to closure has been improved.
+
+Using the `src-cli` to generate diffs and create a campaign plan has been improved:
+
+* It is now easier to continue with a campaign plan for successful patches when other patches fail for some repositories.
+* The dependencies on the external diff and unzip tools have been removed.
+* .gitignore files are now respected when creating patches.
+* On macOS src CLI can [now be installed with Homebrew.](https://github.com/sourcegraph/homebrew-src-cli)
+
+Take a look at the [improved documentation](https://docs.sourcegraph.com/user/campaigns) for usage and examples.
+
+When creating your campaign in Sourcegraph, it is now more clear what state the campaign is in and is easier to differentiate between draft patches and a changeset that has been created on the code host. Additionally, you can now filter the changesets to quickly find which ones need follow up.
+
+Behind the scenes, campaign changesets are now kept up to date using a heuristic solution that reduces the load placed on the code host. Previously, each changeset was updated during regularly scheduled update processes. The new implementation prioritizes syncing changesets based on when last they changed and is also aware of real-time changes that arrive via webhooks.
+
+Code change management campaigns are in private beta. [Watch the campaigns screencasts](https://about.sourcegraph.com/product/code-change-management#see-it-in-action) to see what we have planned, and [apply for early access](https://about.sourcegraph.com/contact/request-code-change-management-demo/) to campaigns for your organization.
+
+## Required migration for Kubernetes deployments
+
+All admins of existing instances with Kubernetes deployments will need to follow the [migration guide for 3.14](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md#314-unreleased). This migration changes the container user from root to non-root for all containers. This change makes Sourcegraph easier to deploy for organizations with strict security requirements. Docker containers and docker-compose deployments are not affected.
+
+## Improved debuggability of Kubernetes deployments
+
+Sourcegraph deployments using docker have had instrumentation available to admins for better monitoring and alerting for several releases. With Sourcegraph 3.14, the Kubernetes cluster deployment now has the same functionality available via the admin page -> Instrumentation or directly at sourcegraph.example.com/-/debug. From there you can trigger pprof profiles, request traces, and more for each server in a Sourcegraph cluster.
+
+## Monitoring and alerting added to the Sourcegraph instrumentation
```

Comment:
@slimsag - please review and make sure I captured this correctly. I had trouble coming up with an intro sentence to lead in about this, will give it more thought in the morning!